src/pages/playerScreen/PlayerScreen.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import ReactPlayer from 'react-player';
+
+import Components from '../../components';
+import { movieService } from '../../services';
+import styles from './playerScreen.module.scss';
+
+const PlayerScreen = () => {
```

Comment:
Player screen can be deleted as we planned to open player in fullscreen mode only